bears/scss/SCSSLintBear.py

```diff
@@ -13,12 +16,295 @@ class SCSSLintBear:
+            Use spaces for indentation (tabs otherwise).
+        :param indent_size:
+            Number of spaces per indentation level.
+        :param leading_zero:
+            Determines whether leading zeros should be written or not in
+            numeric values with a decimal point.
+        :param allow_mergeable_selectors:
+            Allows defining the same selector twice in a single sheet.
+        :param allow_leading_underscore:
+            Allows names to start with a single underscore.
+        :param function_naming_convention:
+            Name of convention (``hyphen``(use lowercase letters and hyphens)
+            (default), ``camel``, ``snake``), or a ``regex`` the name must
+            match (eg: ``^[a-zA-Z]+$``) to use for functions.
+        :param mixin_naming_convention:
+            Name of convention (``hyphen`` (default), ``camel``, ``snake``), or
+            a ``regex`` the name must match (eg: ``^[a-zA-Z]+$``) to use for
+            mixin.
+        :param variable_naming_convention:
+            Name of convention (``hyphen`` (default), ``camel``, ``snake``), or
+            a ``regex`` the name must match (eg: ``^[a-zA-Z]+$``) to use for
+            variable.
+        :param placeholder_naming_convention:
+            Name of convention (``hyphen`` (default), ``camel``, ``snake``), or
+            a ``regex`` the name must match (eg: ``^[a-zA-Z]+$``) to use for
+            placeholder.
+        :param max_depth:
+            Maximum nesting depth.
+        :param use_placeholder_selector_in_extend:
+            Always use placeholder selectors in ``@extend``.
+        :param max_properties:
+            Limit the number of properties in a rule set.
+        :param allow_unit_on_zero_values:
+            Omit length units on zero values.
+        :param check_urls_format:
+            URLs should be valid and not contain protocols or domain names.
+        :param urls_in_quotes:
+            URLs should always be enclosed within quotes.
+        :param allow_unnecessary_parent_reference:
+            Allows use of the parent selector references ``&`` even when they
+            are not unnecessary.
+        :param allow_unnecessary_mantissa:
+            Numeric values can contain unnecessary fractional portions.
+        :param allow_traling_whitespaces:
+            Reports lines containing trailing whitespace.
+        :param allow_traling_semicolon:
+            Property values; ``@extend``, ``@include``, and ``@import``
+            directives; and variable declarations should always end with a
+            semicolon.
+        :param check_imports_path:
+            The basenames of ``@import``ed SCSS partials should not begin with
+            an underscore and should not include the filename extension.
+            These requirements can be modified by changing
+            ``allow_filename_leading_underscore``, and ``allow_extensions``.
+        :param allow_filename_leading_underscore:
```

Comment:
Does this require check_imports_path to be set to true? If yes, it should be mentioned.